src/interface/blas2/backend/default.hpp

```diff
@@ -163,12 +189,24 @@ namespace backend {
 template <uplo_type uplo, transpose_type trn, diag_type diag,
           typename sb_handle_t, typename index_t, typename container_t0,
           typename container_t1, typename increment_t>
-typename sb_handle_t::event_t _tpsv(sb_handle_t& sb_handle, index_t _N,
-                                    container_t0 _mA, container_t1 _vx,
-                                    increment_t _incx,
-                                    const typename sb_handle_t::event_t& _dependencies) {
-  return blas::internal::_tpsv_impl<4, 2, uplo, trn, diag>(sb_handle, _N, _mA,
-                                                           _vx, _incx, _dependencies);
+typename sb_handle_t::event_t _tpsv(
+    sb_handle_t& sb_handle, index_t _N, container_t0 _mA, container_t1 _vx,
+    increment_t _incx, const typename sb_handle_t::event_t& _dependencies) {
```

Comment:
This looks like just a formatting change. Was the old version incorrectly formatted?

Reply:
I used `git-clang-format` without checking. I double checked and yes previously it went over the length went over 80 chars.